repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/keystonev2/selfvalidation/KeystoneV2SelfValidationTest.groovy

```diff
@@ -54,7 +57,10 @@ class KeystoneV2SelfValidationTest extends ReposeValveTest {
     }
 
     def setup() {
+        fakeIdentityV2Service.resetCounts()
         fakeIdentityV2Service.resetDefaultParameters()
+        fakeIdentityV2Service.resetHandlers()
+        sleep(500)
```

Comment:
You mean in Repose 8? I believe it's still needed. If I remember correctly, the work we did was to bypass the Akka cache on retries (like when the admin token expires). In this case, it's not a retry for a single request, but rather behavior across multiple requests, so we don't hit the code that bypasses the cache.